server/lib/dev_tool.ex

```diff
@@ -24,6 +24,10 @@ defmodule DevTool do
       import LenraCommonWeb.ControllerHelpers
       import Plug.Conn
 
+      plug(:put_view, LenraCommonWeb.BaseView)
```

Comment:
I think that it should not be added here but in the router, isn't it ?

You should compare with how it made in the server repo

Reply:
I don't think that this should be in the router. I did not find anything in the server that looked like it.

In my opinion, this should be good to use as is.

Reply:
For your concern about the controller waiting for the webhook's response, we might want to open another issue because this will need to be done on the server & the devtool.